jamovi/surv.r.yaml

```diff
@@ -4,8 +4,8 @@ title: Survival Analysis
 jrs:   '1.1'
 
 items:
-    - name: summary
-      title: Survival Analysis
+    - name:  compsurvTable1
+      title: Events Summary
```

Comment:
why don't we try and keep the names of these tables similar to their titles? we want participants ... sorry, i've been working on experiments all day ... we want *users* to be able to, in RStudio, go:

```
results <- surv(...)
results$ ... <tab>
```

and from the tab list that RStudio suggests, we want them to be able to guess which item corresponds to the table they're interested in.

ideally, your changes would be backwards compatible too, but sometimes that's not possible.

Reply:
That's no problem (it makes a lot of sense !). I've changed it to the original name "summary". I've also changed compsurvTable3 by medianestimates.